hooks/post_gen_project.py

```diff
@@ -39,8 +39,8 @@ def process_license(license_name):
 
     process_license('{{ cookiecutter.license }}')
 
-    if '{{ cookiecutter.use_travis_ci }}' != 'y':
-        remove_file('.travis.yml')
+    if '{{ cookiecutter.use_gh_actions }}' != 'y':
+        remove_dir('.github')
```

Comment:
Just to note that in the future if the template also wants to include GitHub issue/PR templates, this line needs to be revisited to be more specific to just remove the Actions files.

Reply:
ah, good point. i'll fix that.